avm/utilities/pipelines/sharedScripts/Get-NestedResourceList.ps1

```diff
@@ -43,9 +42,10 @@ function Get-NestedResourceList {
         $res += $resource
 
         if ($resource.type -eq 'Microsoft.Resources/deployments') {
-            $res += Get-NestedResourceList -TemplateFileContent $resource.properties.template
-        }
-        else {
+            if ($resource.properties.template -is [System.Collections.Hashtable]) {
```

Comment:
Also in this case, as this is updating utilities and will cause all pipelines to run, I'd kindly ask static validation run and badges for all modules